scripts/local/.env

```diff
@@ -0,0 +1,80 @@
+# -----------------------------------------------------------------------------
+# Required environment variables for local setup
+# -----------------------------------------------------------------------------
+# 1. Locate strings for each section below
+# 2. Paste your strings in the required format, DO NOT add quotation marks
+# 3. Save and run docker-compose up from the scripts/local folder
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# IoTHub connection string:
+# {Your IoT Hub} > Shared access policies > Connection string -- primary key
+# Note: use this value for both environment variables below
+#
+# Example:
+# PCS_IOTHUBREACT_ACCESS_CONNSTRING=HostName=iothub-test123.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
+# PCS_IOTHUB_CONNSTRING=HostName=iothub-test123.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_ACCESS_CONNSTRING=
+PCS_IOTHUB_CONNSTRING=
+
+# -----------------------------------------------------------------------------
+# IoTHub Info can be found in the Azure portal at:
+# {Your IoT Hub} > Endpoints > events
+#
+# Example:
+# PCS_IOTHUBREACT_HUB_NAME=iothub-test123
+# PCS_IOTHUBREACT_HUB_ENDPOINT=Endpoint=sb://iothub-foo-iothub-bar-12345-12345.servicebus.windows.net/;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
+# PCS_IOTHUBREACT_HUB_PARTITIONS=4
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_HUB_NAME=
+PCS_IOTHUBREACT_HUB_ENDPOINT=
+PCS_IOTHUBREACT_HUB_PARTITIONS=
+
+# -----------------------------------------------------------------------------
+# Storage Account information:
+# {Your storage account} > Access keys
+#
+# Example: 
+# PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=storagetest123
+# PCS_IOTHUBREACT_AZUREBLOB_KEY=12345...6789==
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=
+PCS_IOTHUBREACT_AZUREBLOB_KEY=
+
+# -----------------------------------------------------------------------------
+# Storage Endpoint Suffix: 
+# {Your storage account} > Properties > Primary Blob Service Endpoint > value
+# in the URI after the "blob." (eg. `core.windows.net`)
+#
+# Example:
+# PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=core.windows.net
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=
+
+# -----------------------------------------------------------------------------
+# DocumentDb Conn String:
+# {Your Document DB} > Keys > Primary Connection String
+# Note: use this value for all 3 environment variables below
+#
+# Example:
+# PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
```

Comment:
Do we need to add ';' for these specific variables ?

Reply:
I think the point here is to replace everything after = with the conn string you pull from the resource.

Reply:
yes, but only for these variables we are adding ; at the end. This was fixed yesterday in java version in PR https://github.com/Azure/pcs-storage-adapter-java/pull/49/files. We will need to fix it in .net too.

Reply:
Primary connection string value for document db does include a semicolon at the end, so this is correct.